symfony/framework-bundle/3.3/public/index.php

```diff
@@ -23,7 +23,7 @@
 
 // Request::setTrustedProxies(['0.0.0.0/0'], Request::HEADER_FORWARDED);
 
-$kernel = new Kernel(getenv('APP_ENV'), getenv('APP_DEBUG'));
+$kernel = new Kernel($_SERVER['APP_ENV'] ?? 'dev', $_SERVER['APP_DEBUG'] ?? true);
```

Comment:
the behavior has changed. Not setting the env variables was in no-debug mode before. Now, the only way to set it in no-debug mode is to set it to the empty string (which won't work for the console, which expects `'0'`)

Reply:
and `(bool) '0'` is `false`: https://github.com/symfony/symfony/blob/edb8c33f35b981b72adbe23798753703edfe3819/src/Symfony/Component/HttpKernel/Kernel.php#L83

Reply:
my bad, I thought that the only falsy string was the empty one.

At least, this is consistent with `empty('0')`, which also considers it as empty...